tasks/main.yml

```
@@ -79,18 +79,23 @@
     mode: 0770
   become: "{{ deploydrupal_files_become }}"
 
-- name: clear drush cache.
-  shell: "{{ deploydrupal_drush_path }} cache-clear drush"
-  args:
-    chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
-
 - name: clear caches.
   shell: "{{ deploydrupal_drush_path }} cache-rebuild"
   args:
     chdir: "{{ deploydrupal_core_path }}/sites/{{ deploydrupal_site_name }}"
   when:
     - not deploydrupal_site_install
```

```suggestion
    - not deploydrupal_site_install
    - deploydrupal_drush_cache_rebuild
```
Shouldn't we have this here too?

---

We have this on our other one as noted in the other comment thread.

https://github.com/ChromaticHQ/ansible-deploy-drupal/blob/master/tasks/main.yml#L158-L163